folly/EventPortability.cpp

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
Only reason I put it in the root is that it's where the other portability headers will be from #287.